.github/workflows/python-bindings.yml

```diff
@@ -68,7 +69,7 @@ jobs:
 
       - name: Build pyfiction
         working-directory: ${{github.workspace}}
-        run: pip install .
+        run: z3=${{env.ENABLE_Z3}} pip install .
```

Comment:
This looks good. Many thanks! 

Regarding the Python tests: right now, they will fail if Z3 was not enabled. Can we disable the respective ones in that case? I was thinking exposing a certain constant in the `exact` bindings and then checking for it in the Python tests for `exact`. If you know of a better way, please let me know 🙂

Reply:
My idea would be to save the environment variable `z3` in an `.env` file and skip the respective tests based on that value ?

Reply:
If that's possible, that's even better. Thank you!